telemetry/specs/INT.mdk

```diff
@@ -2020,6 +2037,21 @@ INT packet was served.  The calculation mechanism of this value is device
 specific.
 * Queue drop count
   - Total # of packets dropped from the queue
+  
+* Buffer occupancy
+  - The build-up of traffic in the buffer (in bytes, cells, or packets) that the
+INT packet observes in the device while being forwarded. Use case is when buffer is
+shared between multiple queue.
+* Instantaneous buffer occupance
+  - The instantaneous value (in bytes, or cells) of the buffer occupancy the INT
+packet has observed in the device while being forwarded.  The units used need
+not be consistent across an INT domain, but care must be taken to ensure that
+there is a known, consistent mapping of {device, buffer} values to their
+respective unit {bytes, cells}.
+* Average buffer occupancy
+  - The average value (in bytes or cells) of the buffer occupancy 
+the INT packet was observed. The calculation mechanism of this value is device
```

Comment:
grammar: average value of buffer occupancy that the INT packet observed.

Reply:
Added that